app/experimenter/experiments/bugzilla.py

```diff
@@ -105,55 +122,33 @@ def format_creation_bug_body(experiment):
         "description": experiment.BUGZILLA_OVERVIEW_TEMPLATE.format(
             experiment=experiment
         ),
-        "assigned_to": experiment.owner.email,
         "cc": settings.BUGZILLA_CC_LIST,
         "type": "task",
         "priority": "P3",
-        "see_also": [get_bugzilla_id(experiment.data_science_bugzilla_url)],
         "url": experiment.experiment_url,
-        "whiteboard": experiment.STATUS_REVIEW_LABEL,
     }
-    if experiment.bugzilla_tracking_key:
-        bug_data[experiment.bugzilla_tracking_key] = "?"
+
+    if user_exists(experiment.owner.email):
```

Comment:
I would do this in `create_experiment_bug` rather than in here.  We may want to use this format function in some other context and then it would be unexpected for it to make outgoing network calls.  For these optional fields, just pass them in as args to this function.